app/templates/print-overviews/decisions/agendaitems.hbs

```diff
@@ -12,10 +12,15 @@
     @columns={{this.columns}}
     as |agendaitemRow|
   >
-    {{#let (await agendaitemRow.agendaActivity.subcase) (await agendaitemRow.treatment.decisionActivity) as |subcase decisionActivity|}}
+    {{#let
+      (await agendaitemRow.agendaActivity.subcase)
+      (await agendaitemRow.treatment.decisionActivity)
+      as |subcase decisionActivity|
+    }}
       {{#if (and subcase decisionActivity)}}
         <div class="auk-u-m-8">
           <Agenda::Agendaitem::AgendaitemDecision
+            @agendaitem={{agendaitemRow.content}}
```

Comment:
Weird that the model hook returs a promise object instead of a resolved promise, but as already mentioned on other PRs as well, the print-overview pages need a major refactor so going to leave this as it is now.

Reply:
I *think* it's because the component we use here for the table, ember-light-table, wraps all the rows in `ProxyObjects`. So unrelated to the promise hook, but the library's doing. https://github.com/adopted-ember-addons/ember-light-table/blob/master/addon/classes/Row.js#L12